### ISSUE TITLE
apache_conf services return Array objects instead of individual values (e.g. Fixnum)

### ISSUE THREAD

**Issue by davidcpell** [inspec/inspec#1000]
This may be related to #256.
### Description

In my testing for `httpd` I'm finding that the `apache_conf` resource's service checkers are returning arrays rather than single values that I can test against.  The docs for the `be` matcher indicate that I can expect individual values: https://github.com/chef/inspec/blob/master/docs/matchers.rst#be. The `apache_conf` doc also shows comparisons being made with individual values: https://docs.chef.io/inspec_apache_conf.html#matchers (note the doc actually uses an example with `'Timeout'` that is pretty much exactly what I'm trying to do below).

My test looks like this:

```
describe apache_conf do
  its('Timeout') { should eq 60 }
end
```

The error I'm getting is because of not comparing with an Array:

```
  ✖  Apache Config /etc/httpd/conf/httpd.conf Timeout should eq 60 (
     expected: 60
          got: ["60"]

     (compared using ==)
     )
```

I could just change my test to use `["60"]` but the real problem is that I need to use other operators like `<=` which won't work at all with Array.

My current workaround is to hack into the `apache_config` resource using `#params` and then fall back to vanilla RSpec to do the matching, like so:

```
describe 'configuration' do
  let(:config) { apache_conf.params }

  describe 'Timeout' do
    it 'should be <= 300' do
      expect(config['Timeout'].first.to_i).to be <= 300
    end
  end
end
```
### InSpec and Platform Version

Windows 7 Enterprise, SP1
ChefDK 0.17.17
kitchen 1.11.1
### Replication Case
1. Create a cookbook with `$ chef generate cookbook <name>`
2. Add `package 'httpd'` to default recipe
3. Add a test to `test/recipes/default_test.rb` like the one I pasted above
4. `$ kitchen verify`
### Stacktrace

See above for test failure output.

**Comment by chris-rock** [comment]
Thanks for highlighting this issue @davidcpell I assume, you would expect 

```
describe apache_conf do
  its('Timeout') { should eq 60 }
end
```

to work?

**Comment by davidcpell** [comment]
Yes, that's what I was expecting @chris-rock. I should note that I discovered (after writing this issue) that InSpec provides a `cmp` matcher that seems like it could be [used in cases like this](https://github.com/chef/inspec/blob/master/docs/matchers.rst#cmp), but from the documentation it still seems like this is supposed to return a single value rather than an array.
